app/src/main/java/com/example/fivenightsescape/monster/MonsterController.kt

```diff
@@ -11,12 +14,19 @@ private const val DEFAULT_COUNTDOWN_INTERVAL_WANDERING: Long = 1000
 
 private const val DEFAULT_WANDERING_DISTANCE = 10
 
+private const val DEFAULT_ALERT_DISTANCE : Double = 2.0
```

Comment:
Value needs to be higher, with 2 it will play notification when monster is 'inside' player health zone.
In my testing `15.0` looks okay.
![Screenshot 2024-01-22 151408](https://github.com/PKrystian/FiveNightsEscape/assets/97467746/f597eb6c-92b1-4e91-9270-e77a48c96eff)